docs/pages/development.js

```diff
@@ -1,500 +0,0 @@
-// @flow strict
```

Comment:
@rlingineni could we ensure that our `validateAccessibilitySpecs` check still works as expected? Basically we want to ensure every page on our docs site has an accessibility check. In the past we relied on those pages living in `docs/pages`
https://github.com/pinterest/gestalt/blob/master/scripts/validateAccessibilitySpecs.js

I'm also curious as to why we created a specific `markdown` directory, could/should we use `pages` instead? https://nextjs.org/docs/advanced-features/using-mdx

![image](https://user-images.githubusercontent.com/127199/183874795-58981436-e39a-48e9-94f4-2e140f3c2229.png)

It might then also be possible to of the dynamic `docs/pages/[id].js` route and use `MDXProvider` instead.

Reply:
Hi Christian!

Yeah sure! I assume it's `npm run a11y:validate`. I think they might not exist, so let me see what I can do to update this script and make the changes.

For the second thing:

The way pages is structured now in master is a bit different btw from when the PR was made.

Previously, all the files were just flat inside the directory, so it made spotting the `.md` files from the non-markdown files much harder. One of things we wanted is just finding the page we need and editing it quickly from Github.  

But with the new changes, I think it totally makes sense to put them in the same directory structure. And the previous approach could actually end up being more disorganized. I'll look into it and seeing if we can put it in the same directory

Reply:
Okay, so quick update on this. I think the `validateAccessibilitySpecs` check should be passing now. The markdown page is tied to the site index.

For the second part, we're actually using the markdown folder, and not an `.mdx` file alongside the rest of our pages. And this is because we're using`next-mdx-remote` instead of `@next/mdx` as our library. 

MDX Remote allows us to use a wrapper component and distinguish between the js pages and markdown ones. With true MDX, with the regular MDX library, we'd have to write code like:

```
export default ({ children }) => <MyLayoutComponent meta={meta}>{children}</MyLayoutComponent>
```

At the bottom of every MDX file to add an export with a wrapper. One of the goals was to minimize the overhead for writing markdown and going from Dropbox. And there was no generic way to do that.